filter/impl/generic_filter.go

```diff
@@ -99,6 +100,14 @@ func struct2MapAll(obj interface{}) interface{} {
 			}
 		}
 		return result
+	} else if t.Kind() == reflect.Slice {
+		value := reflect.ValueOf(obj)
+		var newTemps []interface{}
```

Comment:
u must set its init len to avoid reallocate its memory. make([]interface{}, 0, value.Len())